Dockerfile

```diff
@@ -1 +1,7 @@
-FROM golang:1.4.1-onbuild
+FROM golang:1.6-alpine
+
+MAINTAINER Tom Denham <tom@tigera.io>
+
+RUN apk -U add bash git musl-dev gcc linux-headers
+WORKDIR /go/src/github.com/coreos/flannel
+
```

Comment:
I'd like to see a Dockerfile that works for cross compiling arm64 builds.
Does alpine have aarch64 cross gcc?  If not, maybe golang:1.6-wheezy would be better.
Does golang:1.6-alpine have arm64 cgo support?